### ISSUE TITLE
sexual impotence

### ISSUE THREAD

**Issue by anna-anagnostop** [mgijax/mammalian-phenotype-ontology#4006]
In [J:346340](https://www.informatics.jax.org/reference/J:346340), Adamtsl2<sup>stb</sup> homozygootes (Genotype [MGI:7628343](https://www.informatics.jax.org/allele/genoview/MGI:7628343)) are reported to exhibit male infertility due to impotence (first animal model of impotence [DOID:1875](https://www.informatics.jax.org/disease/DOID:1875)).

Consider adding impotence under abnormal penile erection MP:0001379 
DEF: persistent or recurrent inability to develop or to maintain a penile erection during sexual activity
SOURCE: [HP:0000802 ](https://www.informatics.jax.org/vocab/hp_ontology/HP:0000802), https://www.ncbi.nlm.nih.gov/books/NBK562253/
SYN: erectile dysfunction
SYN: erectile impotence
SYN: male sexual impotence
SYN: failure of penile erection
SYN: absence of penile erection

Discuss primary term label (impaired penile erection vs impotence vs erectile dysfunction). According to [Britannica](https://www.britannica.com/science/impotence), because of its pejorative connotation  in lay usage and because of confusion about its definition, the word impotence has been eliminated from the technical vocabulary in favour of the term “erectile dysfunction:.

**Comment by anna-anagnostop** [comment]
Most ontologies including DO, HPO, GSSO (Gender, Sex, and Sexual Orientation ontology) and SNOMED just use impotence in the primary label.

**Comment by anna-anagnostop** [comment]
Added new term:
impotence MP:0031646

CHECK: EQ currently uses:
'has part' some ('decreased process quality' and ('characteristic of' some 'penile erection') and ('has modifier' some abnormal)) -- same as the HPO EQ
Used 'decreased process quality' rather than 'absent' to reflect both the inability to develop or to maintain an erection (absent would indicate complete absence of an erection)
